pkg/apis/operations/v1alpha1/imageprepull_types.go

```diff
@@ -61,6 +61,7 @@ type ImagePrePullJobList struct {
 type ImagePrePullJobSpec struct {
 	// ImagePrepullTemplate represents original templates of imagePrePull
 	ImagePrePullTemplate ImagePrePullTemplate `json:"imagePrePullTemplate,omitempty"`
+	ImagePrePullOverrides []ImagePrePullTemplate `json:"imagePrePullOverrides,omitempty"`
```

Comment:
I don't think it's elegant to use template directly in override. If the API is written like this, why don't users just write multiple imagePrepull jobs?